src/rpc/misc.cpp

```diff
@@ -58,6 +58,7 @@ UniValue getinfo(const JSONRPCRequest& request)
             "\nDEPRECATED. Returns an object containing various state info.\n"
             "\nResult:\n"
             "{\n"
+            "  \"deprecation-warning\": \"...\" (string) warning that the getinfo command is deprecated and will be removed in 0.16\n"
```

Comment:
s/0.16/???/ (need to pick some future version)

Reply:
see latest commit